packages/client/hmi-client/src/components/models/tera-model-configuration.vue

```diff
@@ -1,99 +1,153 @@
-							: configurations[0][header].length
-					"
-					:key="i"
-				/>
-			</Row>
-			<Row>
-				<Column v-if="isEditable" selection-mode="multiple" headerStyle="width: 3rem" frozen />
-				<Column :header="isEditable ? 'Select all' : ''" frozen />
-				<Column
-					v-for="(variableName, i) in configurations[0].rates"
-					:header="variableName.target"
-					:key="i"
-				/>
-				<Column
-					v-for="(variableName, i) in configurations[0].initials"
-					:header="variableName.target"
-					:key="i"
-				/>
-				<Column
-					v-for="(variableName, i) in configurations[0].parameters"
-					:header="variableName.id"
-					:key="i"
-				/>
-			</Row>
-			<!--  Add show in workflow later (very similar to "Select variables and parameters to calibrate") -->
-		</ColumnGroup>
-		<Column v-if="isEditable" selection-mode="multiple" headerStyle="width: 3rem" frozen />
-		<Column field="name" frozen>
-			<template #body="{ data, field }">
-				{{ data[field] }}
-			</template>
-			<template #editor="{ data, field }">
-				<InputText v-model="data[field]" autofocus />
-			</template>
-		</Column>
-		<!--TODO: The slice here skips the name attribute, see about skipping it and rates in a clearer way-->
-		<Column
-			v-for="(value, i) of Object.keys(modelConfigurationTable[0]).slice(
-				1,
-				Object.keys(modelConfigurationTable[0]).length
-			)"
-			:key="i"
-			:field="value"
-		>
-			<template #body="{ data, field }">
-				<section class="editable-cell">
-					<span>{{ data[field]?.value }}</span>
-					<Button
-						class="p-button-icon-only p-button-text p-button-rounded p-button-icon-only-small cell-menu"
-						icon="pi pi-ellipsis-v"
-						@click.stop="openValueModal(data, field)"
-					/>
-				</section>
-			</template>
-			<template #editor="{ data, field }">
-				<InputText v-if="data[field].value" v-model="data[field].value" autofocus />
-			</template>
-		</Column>
-		<!-- FIXME: Add checkboxes for calibrate in a seperate PR
-			<ColumnGroup v-if="calibrationConfig" type="footer">
-			<Row>
-				<Column footer="Select variables and parameters to calibrate" />
-				<Column v-for="(name, i) of Object.keys(initialValues[0])" :key="i">
-					<template #footer>
-						<Checkbox v-model="selectedInitials" :inputId="i.toString()" :value="name" />
-					</template>
-				</Column>
-				<Column v-for="(name, i) of Object.keys(parameterValues[0])" :key="i">
-					<template #footer>
-						<Checkbox v-model="selectedParameters" :inputId="i.toString()" :value="name" />
-					</template>
-				</Column>
-			</Row>
-		</ColumnGroup> -->
-	</DataTable>
+		<div class="p-datatable-wrapper">
+			<table class="p-datatable-table p-datatable-scrollable-table editable-cells-table">
+				<thead class="p-datatable-thead">
+					<tr v-if="isEditable">
+						<th class="p-frozen-column"></th>
+						<th class="p-frozen-column second-frozen"></th>
+						<th v-for="({ name, colspan }, i) in tableHeaders" :colspan="colspan" :key="i">
+							{{ capitalize(name) }}
```

Comment:
You can simply use CSS for that with [`text-transform`](https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform)
```css
.p-datatable-thead th {
    text-transform: capitalize;
}
```